lhotse/supervision.py

```diff
@@ -1,14 +1,68 @@
 import logging
 from dataclasses import dataclass
 from itertools import islice
-from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Sequence
+from typing import Any, Callable, Dict, Iterable, List, Mapping, NamedTuple, Optional, Sequence
 
 from lhotse.serialization import Serializable
 from lhotse.utils import Seconds, asdict_nonull, exactly_one_not_null, fastcopy, \
     ifnone, index_by_id_and_check, \
     perturb_num_samples, split_sequence
 
 
+@dataclass(frozen=True, unsafe_hash=True)
+class AlignmentItem(NamedTuple):
+    symbol: str
+    start: Seconds
+    duration: Seconds
+
+    @property
+    def end(self) -> Seconds:
+        return round(self.start + self.duration, ndigits=8)
+
+    def with_offset(self, offset: Seconds) -> 'AlignmentItem':
+        """Return an identical ``AlignmentItem``, but with the ``offset`` added to the ``start`` field."""
+        return fastcopy(self, start=round(self.start + offset, ndigits=8))
+
+    def perturb_speed(
+            self,
+            factor: float,
+            sampling_rate: int,
+    ) -> 'AlignmentItem':
+        """
+        Return a ``AlignmentItem`` that has time boundaries matching the
+        recording/cut perturbed with the same factor. See `perturb_speed()` method
```

Comment:
```
See :meth:`SupervisionSegment.perturb_speed`
```
is better as it will create a cross-reference link to that method in the generated documentation.